Test/git-issues/git-issue-897.dfy

```diff
@@ -0,0 +1,20 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+function missing_number(nums: seq<nat>): nat
+  requires |set i | i in nums| == |nums|
+  requires forall x :: x in nums ==> 0 <= x <= |nums|
+{
+  var p := x => 0 <= x <= |nums| && x !in nums;
+  assert exists x {:trigger p(x)} :: p(x) && forall y {:trigger p(y)} :: p(y) ==> y == x by {
```

Comment:
```suggestion
  assert exists x :: p(x) && forall y :: p(y) ==> y == x by {
```

I view `:trigger` as an advanced feature that shouldn't need to be present in user code. In any case, Dafny picks these triggers for this example, so there's no use in listing them explicitly.

Reply:
OK